LICENSE

```diff
@@ -178,15 +179,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2020 Apache Cordova
```

Comment:
AFAIK this should have all years where changes to this software were made. So in this case, judging from the repo commits, it should be `2016–2020`.

Sorry, I can't make any code suggestions here for some reason.

Reply:
After communicating, I am told year range is not actually recommended. I was told that the year and onward is actually covered.

I will be change these again for all repos but will do them outside of a PR and push them directly to master. Will be doing this most likely during releases.

I am confirming what year I should actually write. Either:
* `2020`, this year as I am now adding it correctly.
* `yyyy`, the year of the first commit in this repo.
* `2012`, the year Cordova graduated in Apache.
* `2011`, the year of Incubation of the Cordova project in Apache.

Reply:
Cool, thanks for doing some research on that